guides/doc-Upgrading_and_Updating/topics/migrating_pulp_content.adoc

```diff
@@ -1,6 +1,6 @@
 [id="migrating_pulp_content"]
 
-= Migrating Content to Pulp 3
+= Preparing to Migrate Content to Pulp 3
```

Comment:
Isn't Preparing Pulp 3 Content Migration better? @melcorr 

Please update the anchor (the id above). Should be in this format ideally: `Preparing_Blah_Blah`